lib/matplotlib/__init__.py

```diff
@@ -578,8 +578,8 @@ def get_configdir():
 
     1. If the MPLCONFIGDIR environment variable is supplied, choose that.
     2a. On Linux, follow the XDG specification and look first in
-        `$XDG_CONFIG_HOME`, if defined, or `$HOME/.config`.
-    2b. On other platforms, choose `$HOME/.matplotlib`.
+        ``$XDG_CONFIG_HOME``, if defined, or ``$HOME/.config``.
+    2b. On other platforms, choose ``$HOME/.matplotlib``.
```

Comment:
Step 5 of this should be removed as we no longer have that code path.

Reply:
done